scripts/util/wait_for_endpoint_debug.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/sh
+
+log () {
+	echo "$(date +'%Y/%m/%d %H:%M:%S') $@"
+}
+
+# If dlv runs as PID 1 in the container it will not properly kill metrictank during shutdown
+# when it receives SIGTERM (docker stop / docker-compose stop / etc...). So, instead we leave
+# this script running in an endless sleep loop and trap SIGTERM, SIGINT, and SIGHUP. Then we can
+# kill both metrictank and dlv and exit this script which will shutdown the container.
+kill_metrictank() {
+  echo "Killing metrictank"
+  pkill metrictank
+  sleep 1
+  echo "Killing dlv"
+  pkill dlv
+  sleep 1
```

Comment:
what's the purpose of the second sleep? i assume the first one is because you want to wait with killing dlv until mt has exited?

Reply:
I don't recall currently. Do you think there is any harm leaving it in?

Reply:
nope, no harm